app/uk/gov/hmrc/gform/models/FastForward.scala

```diff
@@ -62,7 +63,13 @@ object FastForward {
 
   case object Yes extends FastForward
   case class StopAt(stopAt: SectionNumber) extends FastForward
-  case class CYA(from: SectionNumber, to: Option[SectionNumber] = None) extends FastForward // {
+  case class CYA(from: SectionNumber, to: Either[CYA.Summary, SectionNumber] = Left(CYA.FinalSummary))
+      extends FastForward
+  object CYA {
+    sealed trait Summary extends Product with Serializable
+    case object FinalSummary extends Summary
+    case object NonFinalSummary extends Summary
+  }
```

Comment:
Would it be possible to explain in comment what these classes/objects represent?
Is this `FinalSummary`/`NonFinalSummary` task list related or classic form related or both?

Reply:
added the comments to explain which CYA page each variant of Summary represents